Mathlib/Data/Finset/SetLattice.lean

```diff
@@ -0,0 +1,243 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+import Mathlib.Data.Finset.Option
+import Mathlib.Order.Minimal
+
+/-!
+# Lattice operations on finsets
```

Comment:
Please make sure the docstrings of the two files make it clear how to decide which ones to place lemmas in.